tests/playwright/propTests/helperTextAndValidationTextPropType.ts

```diff
@@ -0,0 +1,9 @@
+import type { PropTests } from '../types';
+import { mixPropTests } from '../utils/mixPropTests';
+import { helpTextPropTest } from './helpTextPropTest';
+import { validationTextPropTest } from './validationTextPropTest';
+
+export const helperTextAndValidationTextPropType: PropTests = mixPropTests([
```

Comment:
We talked about following combinations yesterday, should looks like this:

```js
import type { PropTests } from '../types';

export const helperTextAndValidationTextPropType: PropTests = [
  {
    name: 'helpText:string & validationText:undefined',
    props: {
      helpText: 'Some normal helpText.',
      validationText: undefined,
    },
  },
  {
    name: 'helpText:undefined & validationText:string',
    props: {
      helpText: undefined,
      validationText: 'Some normal validationText.',
    },
  },
  {
    name: 'helpText:string & validationText:string',
    props: {
      helpText: 'Some normal helpText.',
      validationText: 'Some normal validationText.'
    }
  }
];
```

I hope that covers alls.